internal/engine/experiment/tor/tor.go

```diff
@@ -319,7 +314,7 @@ func (rc *resultsCollector) measureSingleTarget(
 	}
 	rc.callbacks.OnProgress(percentage, fmt.Sprintf(
 		"tor: access %s/%s: %s", kt.maybeTargetAddress(), kt.target.Protocol,
-		errString(err),
+		failureString(failure),
```

Comment:
The return value is not an `error` anymore but it's `*string` so I changed the name of the `func` handling it.